src/hotspot/cpu/ppc/templateTable_ppc_64.cpp

```diff
@@ -3327,7 +3395,7 @@ void TemplateTable::prepare_invoke(int byte_no,
   // Determine flags.
   const Bytecodes::Code code = bytecode();
   const bool is_invokeinterface  = code == Bytecodes::_invokeinterface;
-  const bool is_invokedynamic    = code == Bytecodes::_invokedynamic;
+  const bool is_invokedynamic    = code == false; // should not reach here with invokedynamic
```

Comment:
This is what I meant.

```suggestion
  const bool is_invokedynamic    = false; // should not reach here with invokedynamic
```

Thanks!